signing/verifier.go

```diff
@@ -0,0 +1,32 @@
+package signing
+
```

Comment:
Where are the tests for this package?

Reply:
For now, it is merely a maatuf. Can't see anything important to test, especially now that the verifier is removed. Let me know if you think otherwise.

Reply:
Added unit tests